doc/tutorials/det/det.txt

```diff
@@ -344,9 +344,9 @@ For further documentation on the DTree class, consult the
 The usual regularized error \f$R_\alpha(t)\f$ of a node \f$t\f$ is given by:
 \f$R_\alpha(t) = R(t) + \alpha |\tilde{t}|\f$ where
 
-\f[
+\f{
```

Comment:
I don't know why this fixed this, or why it was an issue. Not a latex person.

Reply:
If I remember right` \f{` wil not center the formula, really strange that we get a warning here.